tests/hooks/test_hive_hook.py

```diff
@@ -451,7 +452,23 @@ def test_get_results_data(self):
         results = hook.get_results(query, schema=self.database)
         self.assertListEqual(results['data'], [(1, 1), (2, 2)])
 
-    def test_to_csv(self):
+    @unittest.skipIf(NOT_ASSERTLOGS_VERSION < 3.4, 'assertLogs not support before python 3.4')
```

Comment:
I think it is safe to remove this since you should not use python <3.4

Reply:
@Fokko But our CI test will run in python 2.7. If we remove, `test_to_csv` will only test python >= 3.4 and don't cover python < 3.4

Reply:
Ah check, thanks!